package.json

```diff
@@ -8,8 +8,9 @@
     "test": "test"
   },
   "scripts": {
+    "start": "mocha tests --recursive --reporter nyan --watch",
     "test": "mocha tests --recursive --reporter nyan",
-    "start": "mocha tests --recursive --reporter nyan --watch"
+    "test:ci": "mocha tests --recursive"
```

Comment:
I think it's not a big deal having this little repetition here. Keeping things simple stupid has it's benefits too :)